docs/about.rst

```diff
@@ -0,0 +1,31 @@
+About
+=====
+
+Disclaimer
+----------
+
+Naggum don't aim to be yet another Common Lisp or Scheme or whatever
+implementation. Instead, we are trying to deliver a modern Lisp dialect that
+makes use of most of CLI benefits.
+
+Features
+--------
+
+- low level and high level features coupled together
```

Comment:
3\. No, you didn't got it correctly. It just means that it's a simple _machine_-based lisp: we have all _machine_ commands directly available in our syntax, but also (_will_) have a highre-order syntax to organize them in a convenient way (i.e. being a high-level language).

\* by _machine_ I mean CLI VM, of course :)